docs/en/customization/build.md

```diff
@@ -186,7 +184,7 @@ COPY velox.toml .
 ENV CGO_ENABLED=0
 
 # RUN build
-RUN vx build -c velox.toml -o /usr/bin/
+RUN vx build -c plugins.toml -o /usr/bin/
```

Comment:
velox.toml file exists in the repository, this link refers to it: https://github.com/roadrunner-server/velox/blob/master/velox_rr_v2023.toml
(I guess it's better to use `velox_rr_v2023` in this context)